src/coreComponents/constitutive/ExponentialRelation.hpp

```diff
@@ -189,14 +249,17 @@ struct ExponentialCompute< T, ExponentApproximationType::Linear, true >
 /**
  * @class ExponentialRelation
  *
- * Describes a simple exponential relationship with three parameters of the form: \f$ y = y0 * exp(a * (x - x0)) \f$.
+ * Describes a simple exponential relationship with three or four parameters. 
+ * The form of the relationship with three parameters: \f$ y = y0 * exp(a * (x - x0)) \f$.
+ * The form of the relationship with four parameters: \f$ y = y0 * exp(a * (x - x0) + b * (w - w0)) \f$.
  * Optional linear and quadratic Taylor series approximation of the exponent.
- * Implements point and batch direct (y(x)) and inverse (x(y)) compute functions and provides derivatives.
+ * Implements point and batch direct (y(x) and y(x, w)) and inverse (x(y) only for three parameters) compute functions and provides derivatives.
  *
  * @tparam T scalar real-valued type used in computation
- * @tparam EAT the type/order of exponent approximation (linear, quadratic, full)
+ * @tparam EAT the type/order of exponent approximation (linear, quadratic, full, linearThreeVar, quadraticThreeVar, fullThreeVar)
+ * @tparam VAR integer no. of variables (only allows 2 and 3 now), default value is 2
  */
-template< typename T, ExponentApproximationType EAT >
+template< typename T, ExponentApproximationType EAT, integer VAR = 2 >
```

Comment:
TBH I would parametrize this on number of independent (input) variables, i.e. 1 or 2 (rather than 2 or 3). But it's not very consequential, and I want to redesign/simplify the abomination that is `ExponentialRelation` anyway at some point.